StreamChatUITestsAppUITests/Tests/SlowMode_Tests.swift

```diff
@@ -20,7 +20,7 @@ final class SlowMode_Tests: StreamTestCase {
         linkToScenario(withId: 186)
 
         GIVEN("user opens a channel") {
-            backendRobot.setCooldown(enabled: true, duration: 3)
```

Comment:
We probably should increate all tests which use duration "3". Since the failure scenarios might always pass but it could mean the cooldown is already finished. Even better would be to extract the duration to a constant in the test, and use it for every test to keep it consistent 👍

Reply:
Done

Reply:
The change looks good, although the value of slow tests vs having UI tests for this feature is potentially questionable... especially as our UI tests grow in time.

Reply:
@adamrushy are you concerned about the size of our UI test suite?
We leverage parallelisation on GH level, so that engineers are not required to question if +1 automated UI scenario might prolong the CI running times. 

The rule of thumb is to have the features covered by UI tests.